internal/resource/ca_certificate.go

```diff
@@ -159,6 +162,39 @@ func (r CACertificate) Indexes() []model.Index {
 	}
 }
 
+// MarshalResourceJSON is used here to make sure that CA certificate
```

Comment:
nit: May want to make a comment about not saving the certificate metadata in the DB for now, but it's obvious by the code, so works with me.